main.go

```diff
@@ -136,6 +136,9 @@ func processes() {
 }
 
 func processInfo(pid int, period time.Duration) {
+	if period < 0 {
+		log.Fatalf("Cannot process with negative duration: %v", period)
```

Comment:
Maybe rather say something like "Cannot determine CPU usage for negative duration %v" here.

```suggestion
		log.Fatalf("Cannot determine CPU usage for negative duration %v", period)
```

Reply:
Thanks for the review! changed it here https://github.com/google/gops/pull/155/commits/cec8c291a19aaa28a289af62e7ab4a5092c9f4be